cdrom/aulas/urls.py

```diff
@@ -1,8 +1,9 @@
 from django.urls import path
-from aulas.views import AulaDetailView, AulaListView
+from aulas.views import AulaDetailView, AulaListView, AulaCreateView
 
 
 urlpatterns = [
-    path('', AulaListView.as_view(), name='aula_list'),
+    path('aula/', AulaListView.as_view(), name='aula_list'),
```

Comment:
```suggestion
    path('', AulaListView.as_view(), name='aula_list'),
```
Creo que es mejor dejarlo como estaba. Salvo que esto genere algun error